baywatch.install

```diff
@@ -40,3 +41,33 @@ function baywatch_install() {
   $baywatch->enable_bay_platform_dependencies();
   $baywatch->enable_tide_site_theming();
 }
+
+/**
+ * remove any existing nodes of type page content type
```

Comment:
I'm not at all across what this PR is about, but deleting every page node across all tide projects seems quite risky.